src/modules/explorer/actions.js

```diff
@@ -2,6 +2,8 @@ import {createNetworkActionTypes, networkAction} from "../../utils/actions";
 import {jsonRequest} from "../../utils/requests";
 import {extractQueriesFromDataTypeForms} from "../../utils/search";
 
+import FileSaver from "file-saver";
+
 export const PERFORM_SEARCH = createNetworkActionTypes("EXPLORER.PERFORM_SEARCH");
 export const PERFORM_INDIVIDUAL_CSV_DOWNLOAD = createNetworkActionTypes("EXPLORER.PERFORM_INDIVIDUAL_CSV_DOWNLOAD");
```

Comment:
this is now dead code, and it's exported to reducers.js.

Reply:
My bad I forgot to ask about it. This part isn't completed fully yet and requires more discuss about post requests in this case.